webapp/masterclasses.py

```diff
@@ -75,190 +49,41 @@ def index():
         tags=tags,
     )
 
+def format_date(date):
+    # change the date format, like 23 May 2024
+    return date.strftime("%d %b %Y")
 
 def get_upcoming_sessions():
-    try:
-        sheet = get_sheet()
-    except MissingCredential as error:
-        flask.abort(500, str(error))
-
-    SHEET = "Upcoming"
-    RANGE = "A2:F1000"
-    COLUMNS = [
-        ("Topic", str),
-        ("Owner", str),
-        ("Duration", str),
-        ("Date", datetime),
-        ("Notes", str),
-        ("Event", str),
-    ]
-    res = sheet.get(
-        spreadsheetId=SPREADSHEET_ID,
-        ranges=[f"{SHEET}!{RANGE}"],
-        includeGridData=True,
-    ).execute()
-
-    sessions = []
-    for row in res["sheets"][0]["data"][0]["rowData"]:
-        if _has_row_value(row):
-            session = {}
-            for column_index in range(len(COLUMNS)):
-                (column, type) = COLUMNS[column_index]
-                session[column] = get_value_row(
-                    row["values"][column_index]
-                    if index_in_list(row["values"], column_index)
-                    else None,
-                    type,
-                )
-                if COLUMNS[column_index][0] == "Recording":
-                    session["Link"] = get_id(session[column])
-
-            sessions.append(session)
-
-    return sessions
+    session = db_session.query(UpcomingSession).all()
```

Comment:
Should they be ordered by date, same as the previous sessions?

Reply:
I omitted ordering by date as I wanted them to appear in order as pushed in the db. But I believe I'll sort them descendingly, that's a better approach.